config.toml

```diff
@@ -95,15 +99,26 @@ googleAnalytics = ""
     text = '''
 Each BSides is a **community-driven** framework for building events for and by information security community members.  The goal is to expand the spectrum of conversation beyond the traditional confines of space and time.  It creates opportunities for individuals to both present and participate in an intimate atmosphere that encourages collaboration. It is an intense event with discussions, demos, and interaction from participants. It is where conversations for the next-big-thing are happening.
 
-Important Dates:
+## CFP Dates
 
 - Friday, February 1 - CFP opens
 - Thursday, March 7 - CFP closes
 - Friday, March 8 - CFP Review process begins
 - Friday, March 29 - CFP Review process closes
 - Monday, April 1 - CFP Decisions sent out
 
-Ticket availability will be announced soon!
+
+## Ticket Information
+
+We're doing tickets a little differently this year. As always, BSides Atlanta 2019 is a free event that is open to all. This year we've added a Personal Sponsor option. This sponsorship helps pay for putting the conference on and gets you early access to a ticket.
+
+### Ticket Schedule
+
+- Monday, February 18, Noon EST - Personal Sponsor tickets available
+- Friday, March 1, Noon EST - Wave 1 of GA tickets available
+- Monday, April 1, Noon EST - Wave 2 of GA tickets available
```

Comment:
April Fools! I like it on a different day than the first wave.

Reply:
Me too! Hopefully it'll give more people a chance to get the tickets.